_data/compatibility/jaxx.yaml

```diff
@@ -30,7 +30,7 @@ rbf:
         Receiving Replacement Transaction - Replacement transaction shown, original disappears.
 segwit:
   tested:
-    date: "2021-01-27"
+    date: "2019-07-24"
```

Comment:
Just checking, because it's not mentioned in the commit message, was this rollback intended?

Reply:
Yes, @bitschmidty said he was unable to verify https://github.com/bitcoinops/bitcoinops.github.io/pull/510#discussion_r568727862 , so I figured it was easier to drop it in this PR in order to get this PR merged.  Sorry for not mentioning that in the commit message.